### PR TITLE
Add picasa & picasa starter

### PR DESCRIPTION
**Pull request by raisercostin**

### CODE REVIEW

File: picasa.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://en.wikipedia.org/wiki/Picasa",
+    "version": "3.9.141.Build-259",
+    "description": "Picasa is a discontinued image organizer and image viewer for organizing and editing digital photos, plus an integrated photo-sharing website by Google",
+    "license": "Freeware",
+    "hash": "482c1a547d8d3aa25ee446d30ea986de63ef8c8d68b8d1109dd3d9b714e73e08",
+    "url": "https://github.com/raisercostin/raiser-scoop-bucket/releases/download/picasa-3.9.141.259/picasa39-setup-3.9.141.259.exe#dl.7z",
```
Comment (brandon93s) [inline picasa.json]: Why is this pointing to a random release in your repo? Does this need to be in scoop-extras if it's already in your personal bucket?

File: picasa-starter.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "shortcuts": [
+        [
+            "PicasaStarter.exe",
+            "PicasaStarter"
+        ]
+    ],
+    "pre_install":[
+      "If(!(Test-Path \"$persist_dir\\PicasaStarterConfiguration.xml\")) { Set-Content \"$persist_dir\\PicasaStarterConfiguration.xml\" -Value $null }",
+      "If(!(Test-Path \"$persist_dir\\PicasaStarterSettings.xml\")) { Set-Content \"$persist_dir\\PicasaStarterSettings.xml\" -Value $null }",
+      "cmd /c mklink /H $dir\\PicasaStarterConfiguration.xml $persist_dir\\PicasaStarterConfiguration.xml",
```
Comment (Ash258) [inline picasa-starter.json]: Why custom code persist links?? 
What's wrong with 

```json
"persist": ["PicasaStarterConfiguration.xml", "PicasaStarterSettings.xml"]
```

File: picasa-starter.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+    "homepage": "https://sites.google.com/site/picasastartersite/",
+    "version": "2.0",
+    "description": "Picasa is a discontinued image organizer and image viewer for organizing and editing digital photos, plus an integrated photo-sharing website by Google",
+    "license": "Freeware",
+    "hash": "9188940457742451bfe100bf33df3ddf4ff0fa02ce2b242c599f0632a5a9cfbd",
+    "url": "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/picasastarter/PicasaStarter%20V2.0.zip",
+    "bin": [
+      ["PicasaStarter.exe","picasaStarter"],
```
Comment (Ash258) [inline picasa-starter.json]: No need for this rewrite.

File: picasa.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "bin": [
+      ["picasa3.exe","picasa"],
+      ["picasa3.exe","picasa3"],
```
Comment (Ash258) [inline picasa.json]: Irrelevant rewrite

File: picasa.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+      "picasaPhotoViewer.exe"
+    ],
+    "shortcuts": [
+        [
+            "Picasa3.exe",
+            "picasa"
```
Comment (Ash258) [inline picasa.json]: Shortcuts does not look good with first small letter.

No vendors use this capitalization.

Also one shortcut is enough. (Why there is second totally same?)

File: picasa.json (hunk 1 above)
Comment (raisercostin) [reply picasa.json]: The problem is that the picasa is discontinued and the binary is not to be found easily on internet.
I was thinking that there are still people that still use it.
I uploaded a version that shouldn't contain any malware.

File: picasa-starter.json (hunk 2 above)
Comment (raisercostin) [reply picasa-starter.json]: PicasaStarter.exe assumes that the configuration files are in the same folder. I saw that the folders are seen locally to executable but the files not.

File: picasa-starter.json (hunk 3 above)
Comment (raisercostin) [reply picasa-starter.json]: Agree. I'll remove it.

File: picasa-starter.json (hunk 2 above)
Comment (Ash258) [reply picasa-starter.json]: > I saw that the folders are seen locally to executable but the files not.

I don't understand what you said 😶

persist_data function is doing completely same thing: https://github.com/lukesampson/scoop/blob/f284dba5d75ab336d968f6bad719be74261179d2/lib/install.ps1#L1252

File: picasa.json (hunk 4 above)
Comment (raisercostin) [reply picasa.json]: agree. fixed

File: picasa.json (hunk 5 above)
Comment (raisercostin) [reply picasa.json]: agree. fixed

Comment (Ash258) [reply picasa.json]: Mark conversations as resolved.

**Comment by rasa** [comment]
I don’t think we should host dead applications in extras, especially if the developer has yanked the executables. Maybe host them in versions, or another bucket named unsupported, or something like that.

**Comment by moosad** [comment]
> 
> 
> I don’t think we should host dead applications in extras, especially if the developer has yanked the executables. Maybe host them in versions, or another bucket named unsupported, or something like that.

It's okay, but how about `peerblock` then? Is it an exception?

File: picasa-starter.json (hunk 2 above)
Comment (raisercostin) [reply picasa-starter.json]: fixed. thank your for advices.

**Pull request closed without merging**